lib/index.js

```diff
@@ -130,4 +133,87 @@ Dynamoose.prototype.revertDDB = function () {
   this.dynamoDB = null;
 };
 
+Dynamoose.prototype.transaction = async function(items, options, next) {
+  debugTransaction('Run Transaction');
+  const deferred = Q.defer();
+
+  options = options || {};
+  if(typeof options === 'function') {
+    next = options;
+    options = {};
+  }
+
+  if(items === null || items === undefined || items.length === 0) {
```

Comment:
I think we can just do `!Array.isArray(items) || items.length === 0`. This will also cover cases where something that's truthy is passed in. I know this is a bit nitpick-y, but will future-proof.

Reply:
@jutaz Great idea! Will do that soon.

Reply:
Just addressed this in latest commit.